WordPress/Classes/ViewRelated/Tools/HelpshiftPresenter.swift

```diff
@@ -85,7 +88,12 @@ import UIKit
     }
 
     fileprivate var optionsDictionary: [AnyHashable: Any] {
-        return [HelpshiftSupportCustomMetadataKey: HelpshiftUtils.helpshiftMetadata(),
+        var tags: [String]? = nil
```

Comment:
No need to specify nil. An optional will default to a nil value without assignment.  
That said, it looks like `HelpshiftUtils.helpshiftMetadata(withTags:)` could accept an empty array just fine.  Maybe `var tags = [String]()` instead to get rid of the optional?